Terminal.Gui/Views/GraphView/Annotations.cs

```diff
@@ -155,21 +170,25 @@ public LegendAnnotation (Rect legendBounds)
 		/// <param name="graph"></param>
 		public void Render (GraphView graph)
 		{
-			if (Border) {
-				graph.Border.DrawFrame (Bounds, true);
+			if (!_legend.IsInitialized) {
+				_legend.ColorScheme = new ColorScheme () { Normal = Application.Driver.GetAttribute () };
+				graph.Add (_legend);
 			}
 
-			// start the legend at
-			int y = Bounds.Top + (Border ? 1 : 0);
-			int x = Bounds.Left + (Border ? 1 : 0);
+			_legend.Frame = Bounds;
+
+			if (Border) {
+				_legend.OnDrawFrames ();
```

Comment:
Hold on a sec. In the other Issue you said you made the annotation legend a View. But you're not actually making it a SubView of GraphView? That seems really hacky. 

I'd rather you just use this.DrawFrame than jump through the hoops of haivng Legend be a View but not actually be used as a View.

Reply:
See https://github.com/gui-cs/Terminal.Gui/pull/2982#discussion_r1393439023.

Reply:
No, already done. Thanks for the addressed that.